airbyte-integrations/connectors/source-s3/integration_tests/sample_files/csvfile/user_schema.csv

```diff
@@ -0,0 +1,9 @@
+id,name,valid,valid_string,array,dict
```

Comment:
[the test already existed](https://github.com/airbytehq/airbyte/pull/29368/files#diff-cb83352ec27d3b1f8ecfb27d063d2e28eb41aa712183871025840ca571132a6cL19). I should've added this file in a previous PR